mongoengine/document.py

```diff
@@ -291,7 +288,12 @@ def save(self, force_insert=False, validate=True, clean=True,
 
         doc = self.to_mongo()
 
-        created = ('_id' not in doc or self._created or force_insert)
+        # I think the self._created flag is not necessarily required in PyMongo3
```

Comment:
I've just pushed a version commenting the change out, you will see with the new test failures the consequences. It may be right to keep it as it is, but we then need a major change in our save() method on how to handle documents having id when created